app/src/androidTest/java/sw19/moring03/paint/UndoButtonTest.java

```diff
@@ -32,48 +34,43 @@
 
     @Test
     public void testButtonVisibility() {
-        onView(withId(R.id.undoButton)).check(doesNotExist());
         onView(withId(R.id.toolChooserButton)).perform(click());
         onView(withId(R.id.drawPointButton)).perform(click());
         onView(withId(R.id.drawingView)).perform(touchAt(400, 400));
-
+        onView(withId(R.id.undoButton)).check(matches((isEnabled())));
```

Comment:
please check if the undo button is grayed out / disabled before drawing the point